src/commands.ts

```diff
@@ -124,21 +122,26 @@ export class Commands {
 		}
 		const encryptDDO = args[2] === "false" ? false : true;
 		// add some more checks
-		const algoDid = await createAsset(
-			algoAsset.nft.name,
-			algoAsset.nft.symbol,
-			this.signer,
-			algoAsset.services[0].files,
-			algoAsset,
-			this.providerUrl,
-			this.config,
-			this.aquarius,
-			1,
-			this.macOsProviderUrl,
-			encryptDDO
-		);
-		// add some more checks
+		try{ 
+				const algoDid = await createAssetUtil(
+					algoAsset.nft.name,
+					algoAsset.nft.symbol,
+					this.signer,
+					algoAsset.services[0].files,
+					algoAsset,
+					this.providerUrl || this.macOsProviderUrl,
+					this.config,
+					this.aquarius,
+					encryptDDO
+				);
```

Comment:
this is still not OK, we are not able to properly publish to oasis shappire for instance (it publishes but with wrong template and we will not be able to download the asset)... 
as we're always assuming template 1